src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -302,10 +283,9 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     fontSize: normalizeFontSize(24)
   },
-  backbutton: {
-    paddingRight: 16,
-    justifyContent: 'center',
-    paddingLeft: dimen.normalizeDimen(20)
+  backButton: {
+    paddingRight: dimen.normalizeDimen(20),
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `backButton` style seems to be related to the padding on the right side of the button. This should be fine as long as it aligns with your design specifications.



<!-- This is an auto-generated comment by OSS CodeRabbit -->